packages/sources/lotus/src/config/index.ts

```diff
@@ -1,12 +1,22 @@
-import { Requester } from '@chainlink/ea-bootstrap'
-import { Config } from '@chainlink/ea-bootstrap'
+import { AdapterConfig } from '@chainlink/external-adapter-framework/config'
 
-export const NAME = 'LOTUS'
-
-export const DEFAULT_ENDPOINT = 'balance'
-
-export const makeConfig = (prefix?: string): Config => {
-  const config = Requester.getDefaultConfig(prefix, true)
-  config.defaultEndpoint = DEFAULT_ENDPOINT
-  return config
-}
+export const config = new AdapterConfig({
+  API_KEY: {
+    description:
+      'Your Lotus node [API key/token](https://docs.filecoin.io/build/lotus/api-tokens/#obtaining-tokens)',
+    type: 'string',
+    required: true,
+    sensitive: true,
+  },
+  FILECOIN_RPC_URL: {
+    description: 'RPC URL of Filecoin node',
+    type: 'string',
+    default: 'http://localhost:8545',
```

Comment:
I think I see the part of the `json-rpc` adapter this was taken from but I don't think it carries over in the config `lotus` is using. Think we should just mark this as required and remove the default